woocommerce/woocommerce-gateway-stripe
# chore: remove inbox notice when UPE enabled

# Changes proposed in this Pull Request:

The UPE inbox notice is persisted even after UPE has been enabled.
After UPE has been enabled, we should delete the note, so that it's no longer added.

Forgot to add this explicitly to the requirements 😅 

Before:
![2021-09-13 07 56 48](https://user-images.githubusercontent.com/273592/133088384-15e2b640-23f2-451c-8836-c7ac8cf33a5e.gif)


After:
![2021-09-13 08 02 29](https://user-images.githubusercontent.com/273592/133088401-25d886c1-56af-4352-9dee-34fe753c70f8.gif)


# Testing instructions
- Ensure you have UPE disabled
- Ensure you have Stripe enabled
- Ensure you have `_wcstripe_feature_upe_settings` enabled
- Go to the WC home: http://localhost:8082/wp-admin/admin.php?page=wc-admin
- On the inbox note, click "Enable in your store"
- On the wizard, enable UPE
- The inbox notice should be removed from the list of notices
